### PR TITLE
ENH: add two functions for mixed model

### PR DESCRIPTION
**Pull request by HaoLyuKrystal**
Please review and merge this.

### CODE REVIEW

**Comment by boyinggong** [comment]
Please do make style and fix the pep8 style.

**Comment by boyinggong** [comment]
The function aggregate_data only consider the data from aggregate_interval, we should consider the data from aggregate_movement.

File: mousestyles/ultradian/__init__.py [hunk 1]
```diff
@@ -160,6 +164,42 @@ def aggregate_movement(strain, mouse, bin_width):
...
+    with three variables: mouse, strain, feature and hour.
+
+    Parameters:
+    -----------
+    feature: {"AS", "F", "IS", "M_AS", "M_IS", "W"}
+    bin_width: int
+        number of minutes, the time interval for data aggregation
+
+    Returns:
+    -----------
```
Comment (HaoLyuKrystal) [inline mousestyles/ultradian/__init__.py]: what does this return?

File: mousestyles/ultradian/__init__.py [hunk 2]
```diff
@@ -160,6 +164,42 @@ def aggregate_movement(strain, mouse, bin_width):
     return(ts)
 
 
+def aggregate_data(feature, bin_width):
+    """
+    Aggregate all the strains and mouses with any feature together
+    in one dataframe. It will return a dataframe
+    with three variables: mouse, strain, feature and hour.
+
+    Parameters:
```
Comment (HaoLyuKrystal) [inline mousestyles/ultradian/__init__.py]: Fix markup.  It should be:

```
Parameters
----------
```

File: mousestyles/ultradian/__init__.py [hunk 3]
```diff
@@ -160,6 +164,42 @@ def aggregate_movement(strain, mouse, bin_width):
...
+    -----------
+    Column 0: the mouse number (number depends on strain)(0-3)
+    Column 1: the strain of the mouse (0-2)
+    Column 2: hour(numeric values below 24 accourding to bin_width)
+    Column 3: feature values
+
+    Example:
+    -----------
+    test = aggregate_data("W",20)
+
```
Comment (HaoLyuKrystal) [inline mousestyles/ultradian/__init__.py]: This should be a doctests.   For example:
```
>>> test = aggregate_data("W",20)
>>> test.mean()
34.4
```

Comment (HaoLyuKrystal) [inline mousestyles/ultradian/__init__.py]: If it doesn't make sense to provide a little example use case here, then you don't need to include one.

File: mousestyles/ultradian/__init__.py [hunk 4]
```diff
@@ -249,3 +289,57 @@ def strain_seasonal(strain, mouse, feature, bin_width, period_length):
...
+    includes interaction terms while the second model does not include the
+    interaction terms
+
+    Likelihood ratio test p values, if it is below our significance level,
+    we can conclude that the different strains have significantly different
+    time patterns
+
+    Example:
+    ----------
+    mix_strain(data = aggregate_data("W",20),"W")
```
Comment (HaoLyuKrystal) [inline mousestyles/ultradian/__init__.py]: Maybe 
```
>>> mix_strain(data = aggregate_data("W",20),"W")
0.02
```

**Pull request merged**